mysql/run-benchmark.sh

```diff
@@ -0,0 +1,131 @@
+
+  echo "Running benchmark"
+  sysbench $PARAMS --db-driver=mysql --mysql-user=root --tables=16 --table-size=10000 --mysql-db="$MYSQL_DATABASE" --time=5 --report-interval=1 --verbosity=5 --rate=40 --events=0 /tmp/output_json.lua run > /tmp/output-$I.txt
+
+  echo "Running cleanup"
+  sysbench $PARAMS --db-driver=mysql --mysql-user=root --tables=16 --table-size=10000 --mysql-db="$MYSQL_DATABASE" /usr/local/share/sysbench/oltp_read_write.lua cleanup
+
+  # General Statistics
+  total_time=$(cat /tmp/output-$I.txt | grep -A 3 "General statistics" | grep "total time:" |cut -d":" -f2| xargs | cut -d "s" -f1)
+  total_events=$(cat /tmp/output-$I.txt | grep -A 3 "General statistics" | grep "total number of events:" |cut -d":" -f2 |xargs)
+
+  # Latency
+  latency_min=$(cat /tmp/output-$I.txt | grep -A5 "Latency (ms)" | grep "min:" |cut -d":" -f2 |xargs)
+  latency_max=$(cat /tmp/output-$I.txt | grep -A5 "Latency (ms)" | grep "max:" |cut -d":" -f2 |xargs)
+  latency_avg=$(cat /tmp/output-$I.txt | grep -A5 "Latency (ms)" | grep "avg:" |cut -d":" -f2 |xargs)
+  latency_percentile=$(cat /tmp/output-$I.txt | grep -A5 "Latency (ms)" | grep "percentile:" |cut -d":" -f2 |xargs)
+  latency_sum=$(cat /tmp/output-$I.txt | grep -A5 "Latency (ms)" | grep "sum:" |cut -d":" -f2 |xargs)
+
+  # Thread fairness
+  thread_fairness_events=$(cat /tmp/output-$I.txt | grep -A2 "Threads fairness" | grep "events" |cut -d":" -f2 |xargs | cut -d "/" -f1)
+  thread_fairness_execution_time=$(cat /tmp/output-$I.txt | grep -A2 "Threads fairness" | grep "execution time" |cut -d":" -f2 |xargs | cut -d "/" -f1)
+
+  # JSON data extracted from txt file and converted to json
+  sed -n '18,/Time limit exceeded, exiting.../p' /tmp/output-$I.txt | sed 's/Time limit exceeded, exiting...//' | sed '${s/$/]/}' > output-$I.json
+
+  cat <<EOF | curl --data-binary @- $PUSHGATEWAY_URL/mysql-$I
+
+    # TYPE general_statistics_total_ gauge
+    $(echo "general_statistics_total_time ${total_time}")
+    $(echo "general_statistics_total_events ${total_events}")
+
+    # TYPE latency_ gauge
+    $(echo "latency_min ${latency_min}")
+    $(echo "latency_max ${latency_max}")
+    $(echo "latency_avg ${latency_avg}")
+    $(echo "latency_percentile ${latency_percentile}")
+    $(echo "latency_sum ${latency_sum}")
+
+    # TYPE thread_fairness_ gauge
+    $(echo "thread_fairness_events ${thread_fairness_events}")
+    $(echo "thread_fairness_execution_time ${thread_fairness_execution_time}")
+
+    # TYPE run_time_information_all gauge
+    $(cat /tmp/output-$I.json | jq -r '.[] | "run_time_information_all{time="+ "\"" + (.time|tostring) + "\"" + ",threads=" + "\"" + (.threads|tostring) + "\"" + ",latency=" + "\"" + (.latency|tostring) + "\"" + ",errors=" + "\"" + (.errors|tostring) + "\"" + ",reconnects=" + "\"" + (.reconnects|tostring) + "\"" +  "}" + " " + (.tps|tostring)')
```

Comment:
Add lables to identify these metrics on the basis of the cluster and machine type. As it is done in pytorch. But adding plain `instance` can mean conflict with existing `instance` label. 
![Screenshot from 2021-04-30 18-31-43](https://user-images.githubusercontent.com/5815795/116698757-5e9d4400-a9e2-11eb-85da-0dfa1e401857.png)